app/javascript/chat/ChatChannelSettings/MembershipManager/InvitationLinkManager.jsx

```diff
@@ -0,0 +1,127 @@
+import { h, Component } from 'preact';
+import PropTypes from 'prop-types';
+import { Button } from '@crayons';
+
+function isNativeAndroid() {
```

Comment:
I've seen these in a bunch of places in the code base. Consider importing them from a module so that we avoid copy pasta. For the time being import it only for stuff relevant to this PR and for other parts of the code base, it can be a follow up PR.